Symfony/CS/Tests/Fixer/Contrib/AlignEqualsFixerTest.php

```diff
@@ -31,6 +31,37 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                $a        = 1;
```

Comment:
Nope. This case is the case "Do not touch this!".

I just followed **your** docs: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.10/COOKBOOK-FIXERS.md#keeping-things-as-they-are :-)